config.go

```diff
@@ -13,7 +13,8 @@ type Config struct {
 // CORSConfig headers configuration.
 type CORSConfig struct {
 	// AllowedOrigin: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Access-Control-Allow-Origin
-	AllowedOrigin string `mapstructure:"allowed_origin"`
+	AllowedOrigin         string `mapstructure:"allowed_origin"`
+	AllowedOriginWildcard string `mapstructure:"allowed_origin_wildcard"`
```

Comment:
`AllowedOriginRegex` will be more clear? @rustatian WDYT?

Reply:
Hmm... Yeah, `AllowedOriginRegex` will be clearer to understand. Good take :+1: